### PR TITLE
test: add php 8 versions to travis

### PR DESCRIPTION
**Pull request by Chris53897**
hi,
this PR tries to add tests for php 8
i do not use Travis. Please check if this is working.

ruflin/elastica:6 is now supporting php8
https://packagist.org/packages/ruflin/elastica
https://packagist.org/packages/ruflin/elastica#6.2.0

Greetz

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,11 +30,6 @@ matrix:
      # no flag build with code coverage:
      - php: 7.4
        env: ENABLE_CODE_COVERAGE="yes"
-     # --ignore-platform-reqs for ruflin/elastica:6 which doesn't support PHP8
-     - php: 8.0
-       env: COMPOSER_FLAGS="--ignore-platform-reqs"
-     - php: nightly
```
Comment (maxhelias) [inline .travis.yml:34]: We can/need keep nightly stage

File: .travis.yml [hunk 2]
```diff
@@ -42,12 +39,12 @@ matrix:
...
          - SYMFONY_DEPRECATIONS_HELPER=weak
-     - php: 8.0snapshot
+     - php: 8.0
        env:
-         - DEPENDENCIES=dev
-         - COMPOSER_FLAGS="--ignore-platform-reqs"
-    allow_failures:
-     - php: 8.0snapshot
+         - SYMFONY_DEPRECATIONS_HELPER=weak
+     - php: 8.1
```
Comment (maxhelias) [inline .travis.yml:45]: no need to specify again php 8.0 and 8.1, just remove 8.0snapshot

**Comment by Chris53897** [comment]
@maxhelias Thanks for the comments. I did the changes. But i am not sure if i understood you correct. Please have a look if i did this correct.

**Review by maxhelias — APPROVED** [review]

**Comment by curry684** [comment]
Dropped Travis and migrates to Github Actions with only PHP 8.0+, thank you.

**Pull request closed without merging**